testing/common.go

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (C) 2017 XLAB, Ltd.
+ *
+ * This work is open source software, licensed under the terms of the
+ * BSD license as described in the LICENSE file in the top-level directory.
+ */
+
+package testing
+
+import (
+	"strings"
+)
+
+// FixIndent moves the inline yaml content to the very left.
```

Comment:
May I suggest making this change part of a separate commit/PR? Plenty of changes are only because of the fact this utility func has been moved outside of the existing package.